### ISSUE TITLE
Please can you add Hierarchical model for MMM 

### ISSUE THREAD

**Issue by kurvaraviteja355** [pymc-labs/pymc-marketing#334]

**Comment by ricardoV94** [comment]
@kurvaraviteja355 can you expand a bit on your request? We have an issue for adding hierarchical structure on advertisement channels https://github.com/pymc-labs/pymc-marketing/issues/88

Is that what you have in mind?

**Comment by kurvaraviteja355** [comment]
Yes, I'm referring to the same thing. For example, I have a dataset with various nations and multiple channels (some of which are hierarchical), and I need to determine the contribution of each channel and ROAS in each country. As an example, instead of one model for the country, develop a single hierarchical model.  This also aids in learning from similar countries if any of the countries lacked sufficient data points.

**Comment by williambdean** [comment]
Closing this following the merge of https://github.com/pymc-labs/pymc-marketing/pull/1036

Give it a try and please open another issue if needed
